src/inference/include/openvino/runtime/remote_context.hpp

```diff
@@ -26,6 +26,10 @@ class Core;
 
 namespace ov {
 
+namespace autobatch_plugin {
+class RemoteContext;
+}
+
```

Comment:
I don't think that you need a friend here.

Reply:
> I don't think that you need a friend here.

@ilyachur proxy plugin also used friendly class. https://github.com/ilyachur/openvino/blob/e5f99f0311152f62bf4a7a611b96f05c2d001727/src/inference/include/openvino/runtime/remote_context.hpp#L75
If no friendly class, how can I get the 'ov::IRemoteContext' from 'ov::RemoteContext'

Reply:
Proxy plugin have a friend only because it emulates behaviour of low-level plugin.

YOu don't need to have this friend, if auto batch plugin will wrap hardware low-level plugin.